### PR TITLE
UK gov

### PR DESCRIPTION
**Pull request by abrudz**
Per UK government's bullshit list: https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#words-to-avoid

### CODE REVIEW

File: src/terms.js [hunk 1]
```diff
@@ -232,30 +240,32 @@ export default [
...
-    'incentivi(s|z)e',
+    '(dis)?incentivi(s|z)e',
     'increase the odds',
     'incredibl(e|y)',
     '(inflat|redeem)able value',
     'influencers?',
     'influx',
+    'initiat(es?|ing|ives?)',
     'innovat(e|ed|ion|ive|ing|or)s?',
+    'in order',
```
Comment (mourner) [inline src/terms.js:255]: Might be too generic — e.g. in a sentence with `in order to`, replacing with "bullshit" will make the sentence nonsensical.

File: src/terms.js [hunk 2]
```diff
@@ -135,6 +140,7 @@ export default [
     'downsi(s|z)e',
     'drill down',
     'drink the kool-aid',
+    'drive( out|s)?'.
```
Comment (mourner) [inline src/terms.js:143]:
```suggestion
    'drive( out|s)?',
```

**Review by mourner — APPROVED** [review]
This is great! Let's fix the typo and make sure the tests are green.

File: src/terms.js (hunk 1 above)
Comment (abrudz) [reply src/terms.js:255]: Yeah, what the government means is to simply erase the words "in order". This kind of change may not be appropriate for bullshit.js

**Comment by mourner** [comment]
@abrudz ESLint complains about some trailing whitespace which needs to be removed for a green build, but otherwise this looks good!

**Comment by abrudz** [comment]
I don't see any trailing whitespace. Would appreciate help.

File: src/terms.js [hunk 3]
```diff
@@ -441,12 +457,14 @@ export default [
     'sticky-?ness',
     'strateg(y|ic|ize|ise)',
     'streamlin(ed|e|ing)',
+    'strength(en(ing)?|s)?',
```
Comment (mourner) [inline src/terms.js:460]:
```suggestion
    'strength(en(ing)?|s)?',
```

**Review by mourner — COMMENTED** [review]
Found it! The [Travis log](https://travis-ci.com/github/mourner/bullshit.js/builds/213521387) shows the line numbers.

**Pull request merged**